decentraland/unity-renderer
# feat: Add a loading spinner in the taskbar while the Friends feature is being initialized

## What does this PR change?
Fix #1748

![image.png](https://images.zenhubusercontent.com/5eb2b8ca65c88f6e391ef16e/c66b85d1-ef4c-40d3-bd55-c3b61029087d)

**Figma:** https://www.figma.com/file/SxxomcIZI1zyytMZmFJAvS/Social-Module-%5Bwip%5D?node-id=604%3A13360

**Acceptance Criteria**
- [x] Remove Friends connection fail notification
- [x] While the Friends panel is loading, a loading spinner will be showed.
- [x] We won't never contemplate a "failed" status of the friends feature because we maked it be indefinitely retried from kernel.

**NOTE:** Use `feat/SocialBar` as base branch.

## Kernel side
https://github.com/decentraland/kernel/pull/282

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/improve-the-way-friends-feature-fails-explorer-side&kernel-branch=feat/improve-the-way-friends-feature-fails-kernel-side
2. Enter the world and wait for friends initialization.
3. Notice that a loading spinner is showed in the taskbar while friends are not initialized.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
